build.gradle

```diff
@@ -96,6 +96,12 @@ allprojects { subproj ->
 		mavenCentral()
 		maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
 	}
+	buildscript {
+		repositories {
+			// mavenLocal()
+			mavenCentral()
+		}
+	}
```

Comment:
[I will have a look](https://github.com/diffplug/spotless/issues/120#issuecomment-307898092) into this subject and hopefully provide a fix for future versions of _spotless_. But I would like to keep this out of this PR is you agree.